tool/tctl/common/helpers_test.go

```diff
@@ -0,0 +1,115 @@
+package common
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/gravitational/teleport/lib/config"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/teleport/lib/utils"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func runResourceCommand(t *testing.T, fc *config.FileConfig, args []string, ) (*bytes.Buffer, error) {
+	var stdoutBuff bytes.Buffer
+	command := &ResourceCommand{
+		stdout: &stdoutBuff,
+	}
+	cfg := service.MakeDefaultConfig()
+
+	app := utils.InitCLIParser("tctl", GlobalHelpString)
+	command.Initialize(app, cfg)
+
+	selectedCmd, err := app.Parse(args)
+	require.NoError(t, err)
+
+	var ccf GlobalCLIFlags
+	ccf.ConfigString = mustGetBase64EncFileConfig(t, fc)
+
+	clientConfig, err := applyConfig(&ccf, cfg)
+	require.NoError(t, err)
+
+	client, err := connectToAuthService(context.Background(), cfg, clientConfig)
+	require.NoError(t, err)
+
+	_, err = command.TryRun(selectedCmd, client)
+	if err != nil {
+		return nil, err
+	}
+	return &stdoutBuff, nil
+}
+
+func mustDecodeJSON(t *testing.T, r io.Reader, i interface{}) {
+	err := json.NewDecoder(r).Decode(i)
+	require.NoError(t, err)
+}
+
+func mustGetFreeLocalListenerAddr() string {
```

Comment:
Make this consistent with other mustXXX methods here and have this accept `t *testing.T` and use `require` instead of panic?